doc/user/user/pipelines/index.md

```diff
@@ -337,11 +341,8 @@ For example, you may encounter an error when running the "Assembly and Annotatio
 
 ![analyses-table-error](images/analyses-table-error.png)
```

Comment:
This image should be redone with a scaled down browser.  It's too big to see what it's trying to show.

Reply:
Updated in 6d265b032635f8e0f76f98eddb88a30e899bf36f